src/services/date-service.ts

```diff
@@ -1,3 +1,7 @@
+export const timeStringFromDate = (date): string => {
+  const [HH, MM] = new Date(date).toLocaleTimeString().split(':');
```

Comment:
Vet du at du alltid får `date` på lik måte? Kan kanskje ha en sjekk på regex og returner date dersom det ikke matcher.

Reply:
Ville kanskje heller gått for noe som dette, da du slipper å gjøre streng-parsing: 
```suggestion
  const date = new Date(dateString);
  return `${date.getHours()}:${date.getMinutes()}`;
```
Du kan ev. left-padde dem med 0 hvis du trenger det slik:

```ts
const leftPad = (input: number): string => input < 10 ? `0${input}` : input.toString();
```

Reply:
Poenget var å få med left padding. Derfor jeg gjorde det slik, men løsningen din var fin!